### PR TITLE
Version 2.0.0: new approach to replace the font families

### PR DESCRIPTION
**Pull request by matthijs110**
Thanks to @luuk777w version 1.3.0 uses a new approach to set the 'Twemoji Country Flags' as primary font. From now on, the extention creates an additional style tag in the document head containing **each** exisiting CSS selector of the website that has a `font-family` property inside it. This way, the font family is changed on class base, instead of individual HTML elements. 

This PR also fixes issue #2 where - in rare occasions - the font family could not be set.

Last, but not least, this version should also improve the experience on YouTube. Sometimes the flag was not visible at all.

### CODE REVIEW

File: src/content.js [hunk 1]
```diff
@@ -4,56 +4,106 @@ const replacementFontName = "Twemoji Country Flags";
 // Some elements can be ignored.
 const ignoredElements = ["style", "script", "svg"];
```
Comment (luuk777w) [inline src/content.js:5]: I believe this line can now be removed

File: src/manifest.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "name": "Country Flag Fixer",
     "default_locale": "en",
     "description": "__MSG_extDesc__",
-    "version": "1.2.5",
+    "version": "1.3.0",
```
Comment (luuk777w) [inline src/manifest.json:5]: Maybe upgrade to v2.0.0? Since it is a major change, and not minor in my opinion

**Review by luuk777w — CHANGES_REQUESTED** [review]

File: src/manifest.json (hunk 2 above)
Comment (matthijs110) [reply src/manifest.json:5]: Good point, I think we can see this change as major, even though it functionally does not change.

File: src/content.js (hunk 1 above)
Comment (matthijs110) [reply src/content.js:5]: You are right!

**Review by luuk777w — APPROVED** [review]

**Pull request merged**